app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -240,6 +244,21 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
         return settings().lastKnownMode
     }
 
+    /**
+     * Determines whether the activity should be pushed to be backstack (i.e., 'minimized' to the recents
+     * screen) upon starting.
+     * @param intent    The intent that started this activity. Is checked for having the 'START_IN_RECENTS_SCREEN'-extra.
+     * @return          True if the activity should be started and pushed to the recents screen, false .
+     */
+    private fun shouldStartInRecentsScreen(intent: Intent?): Boolean {
+        intent?.toSafeIntent()?.let {
+            if (it.hasExtra(START_IN_RECENTS_SCREEN)) {
```

Comment:
Is the `if (it.hasExtra(START_IN_RECENTS_SCREEN))` necessary? Your `getBooleanExtra` will return false as default